courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<p>It would be nicer (i.e., simpler to use and less error-prone) to the client code if the ".encode('utf-8')" was taken care of in the body of "utils_preproc.preprocess_sentence".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>